string type
string name
int32 id
int32 sampling_rate
---
